kolibri/content/migrations/0001_initial.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
```

Comment:
It's unavoidable, we have to make migrations to run the tests unfortunately. But we can squash the migrations prior to a release.